index.html

```diff
@@ -0,0 +1,219 @@
+                <img class="img__class" id="float__img" src="./img/pic2.png" alt="">  
+            </div>    
+            <div class="right__box" id="uber_box">
+                <h1 class="bold-text" id="text__3">Multi-Post Stories</h1>
+                <h1 class="bold-text" id="text__4">Uber Navigation</h1>
+                <ul class="selections">
+                    <li class="main__txt__wrapper"><a href="#" class="main__txt"></a>Uber</li>
+                    <li class="circle__wrapper"><a href="#" class="circle"></a></li>
+                    <li class="list__wrapper"><a href="#" class="list">LEAD DEVELOPER</a></li>
+                    <li class="circle__wrapper"><a href="#" class="circle"></a></li>
+                    <li class="list__wrapper"><a href="#" class="list">2015</a></li>
+                </ul>
+                <p class="signup-disc">A daily selection of privately personalized reads: no acounts or signup's required</p>
+                <ul class="platform">
+                    <li class="pla__btn">html</li>
+                    <li class="pla__btn">css</li>
+                    <li class="pla__btn">javascript</li>
+                </ul>
+                <button class="see-project">See Project</button>
+            </div>    
+        </div>
+
+    </section>
+    <section class="about-section">
+        <div class="about__wrapper">
+            <div class="discription__wrapper">
+                <div class="details-about-me" id="about_title">
+                    <h1 class="bold-text" >About Myself</h1>
+                    <p class="signup-disc" id="about_detail">i am a software developer by profesion and i help develop many softwares, you can go ahed and check out some of my projects in my portofolio, both mobile and web based and continuing to do so. if you need help regarding software development or have a project in mind feel free to contact me.</p>
+                    <h2 class="let-connect" id="lets-connect_title">LET'S CONNECT</h2>
+                </div>
+                <ul class="contact_us_wrapper">
+                    <li><a class="icon" href="https://github.com/addisalemseifu"><i class="fab fa-github-square"></i></a></li>
+                    <li><a class="icon" href="https://www.linkedin.com/in/addisalemseifu/"><i class="fab fa-linkedin-in"></i></a></li>
+                    <li><a class="icon" href="https://github.com/addisalemseifu"><i class="fab fa-angellist"></i></a></li>
+                    <li><a class="icon" href="https://twitter.com/SeifuAddisalem"><i class="fab fa-twitter"></i></a></li>
+                    <li><a class="icon" href="https://github.com/addisalemseifu"><i class="fab fa-medium-m"></i></a></li>  
+                </ul>
+                <button class="see-project" id="project_about">See Project</button>
+            </div>
+            <div class="performance__wrapper">
+                <div class="performance">
+                    <h3 class="languages-title">Languages</h3>
```

Comment:
Please add a `down arrow` on the very right side of `Languages` as it can be seen in the design.